backend/src/main/java/com/swingevents/SwingEvents/SwingEventsApplication.java

```diff
@@ -1,5 +1,8 @@
 package com.swingevents.SwingEvents;
 
+import lombok.extern.slf4j.Slf4j;
```

Comment:
If you already use lombok you can leverage this functionality:
https://projectlombok.org/api/lombok/extern/slf4j/Slf4j.html